pom.xml

```diff
@@ -209,7 +213,7 @@
 	</dependencies>
 	<url>https://github.com/WorksApplications/elasticsearch-sudachi</url>
 	<description>The Japanese analysis plugin for elasticsearch</description>
-	<inceptionYear>2017</inceptionYear>
+	<inceptionYear>2017-2018</inceptionYear>
```

Comment:
According to official document, `inceptionYear` should be 4 digits.

http://maven.apache.org/ref/3.5.3/maven-model/maven.html

Reply:
Thanks! I'd like to revise inceptionYear to 2017.